test/versioned/otel-bridge/span.test.js

```diff
@@ -394,3 +395,40 @@ test('Otel producer span test', (t, end) => {
     })
   })
 })
+
+test('messaging consumer metrics are bridged correctly', (t, end) => {
+  const { agent, tracer } = t.nr
+  const attributes = {
+    [ATTR_MESSAGING_SYSTEM]: 'kafka',
+    [ATTR_MESSAGING_OPERATION]: 'getMessage',
+    [ATTR_SERVER_ADDRESS]: '127.0.0.1',
+    [ATTR_MESSAGING_DESTINATION]: 'work-queue',
+    [ATTR_MESSAGING_DESTINATION_KIND]: 'queue'
+  }
+
+  tracer.startActiveSpan('consumer-test', { kind: otel.SpanKind.CONSUMER, attributes }, (span) => {
+    const tx = agent.getTransaction()
+    const segment = agent.tracer.getSegment()
+    assert.equal(segment.name, 'OtherTransaction/Message/kafka/queue/Named/work-queue')
+    span.end()
+    const duration = hrTimeToMilliseconds(span.duration)
+    assert.equal(duration, segment.getDurationInMillis())
+    tx.end()
+
+    assert.equal(tx.type, 'message')
+
+    const unscopedMetrics = tx.metrics.unscoped
+    const expectedMetrics = [
+      'OtherTransaction/all',
+      'OtherTransaction/Message/all',
+      'OtherTransaction/Message/OtherTransaction/Message/kafka/queue/Named/work-queue',
```

Comment:
these names will need to be updated once you fix the suggestions in the consumer synthesizer. it shouldn't have `OtherTransaction/Message` twice